### PR TITLE
Add variables for Neumann boundary conditions to elliptic solutions

### PR DESCRIPTION
**Pull request by nilsvu**
## Proposed changes

These will be used to impose Neumann-type boundary conditions from the analytic solutions.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `major new feature` if appropriate.

### CODE REVIEW

File: src/PointwiseFunctions/AnalyticSolutions/Elasticity/CMakeLists.txt [hunk 1]
```diff
@@ -28,6 +28,7 @@ target_link_libraries(
   ConstitutiveRelations
   DataStructures
   Elasticity
+  ElasticPotentialEnergy
```
Comment (kidder) [inline src/PointwiseFunctions/AnalyticSolutions/Elasticity/CMakeLists.txt:30]: should come before Elasticity

**Review by kidder — CHANGES_REQUESTED** [review]
you can squash immediately

**Review by fmahebert — DISMISSED** [review]

**Comment by nilsvu** [comment]
Rebased and squashed. Thanks for your reviews @kidder and @fmahebert!

**Review by kidder — DISMISSED** [review]

**Review by fmahebert — DISMISSED** [review]

**Comment by nilsvu** [comment]
Rebased on blaze upgrade

**Review by fmahebert — APPROVED** [review]

**Review by kidder — APPROVED** [review]

**Pull request merged**